src/app/containers/create-movie/displaySuccess.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState } from 'react';
+
+const DisplaySuccess = ({ data }) => {
+  const [visible, setVisible] = useState(true);
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      setVisible(false);
+    }, 5000);
+
+    return () => clearTimeout(timer);
+  }, []);
+
+  if (!visible) return null;
+
+  return (
+    <div className="absolute bottom-0 bg-green-100 border border-green-500 text-green-700 rounded p-4 w-full max-w-md text-center mt-4">
+      <h3 className="font-bold mb-2">Newly Created Movie</h3>
+      <p><strong>ID:</strong> {data.id}</p>
```

Comment:
We could destructure id and name from data since the object data itself it's not used

Reply:
solved at [6c63366](https://github.com/OctaFloare/flitnext-2024/pull/12/commits/6c63366ce437096d199fa5b5aafd3880fbd91fc9)